tools/autograd/gen_autograd.py

```diff
@@ -44,13 +44,15 @@
     'expand': 'self',
     'narrow': 'self',
     'permute': 'self',
+    'reshape_as': 'self',
```

Comment:
I think reshape_as is the same as reshape here and should be moved below.

Reply:
Done.